sweep/sweeper.go

```diff
@@ -188,6 +202,9 @@ type pendingInput struct {
 	// lastFeeRate is the most recent fee rate used for this input within a
 	// transaction broadcast to the network.
 	lastFeeRate chainfee.SatPerKWeight
+
+	// rbf records the RBF constraints.
```

Comment:
Q: Whats the name of this new design pattern we are using here with the generic design ?

Reply:
You mean the whole sweeper?